InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs

```diff
@@ -0,0 +1,24 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using FluentAssertions;
+using Xunit;
+namespace InternTrack.Core.Api.Tests.Acceptance.Apis.Home
+{
+    public partial class HomeApiTests
+    {
+        [Fact]
+        public async Task ShouldReturnHomeMessageAsync()
+        {
+            // Given
+            string expectedMessage = 
+                "Thank you, Mario. But the princess is in another castle!";
+            // When
```

Comment:
New line before `// when` and make sure you lower case all letters `// given` and `// when` and `// then`

Reply:
Do the same for `// then` need new line